src/handlers.rs

```diff
@@ -395,6 +402,42 @@ fn send_timer_ready(d: *const DenoC, timer_id: u32, done: bool) {
   );
 }
 
+fn handle_make_temp_dir(
+  d: *const DenoC,
+  builder: &mut FlatBufferBuilder,
+  dir: Option<&str>,
+  prefix: Option<&str>,
+  suffix: Option<&str>,
+) -> HandlerResult {
+  let deno = from_c(d);
+  if !deno.flags.allow_write {
+    let err = std::io::Error::new(
+      std::io::ErrorKind::PermissionDenied,
+      "allow_write is off.",
+    );
+    return Err(err.into());
+  }
+  // TODO(piscisaureus): use byte vector for paths, not a string.
+  // We can't assume that paths are always valid utf8 strings.
```

Comment:
Could you create an issue for this and link to it here? This is a bigger issue than just makeTempDir.

Reply:
done